app/build.gradle

```diff
@@ -46,6 +46,10 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
     testCompile "org.robolectric:robolectric:3.3"
+    testCompile group: 'org.robolectric', name: 'shadows-support-v4', version: '3.3.1'
```

Comment:
I merged this build.gradle into master by Taimoor's request. There are no conflicts with this branch and master.